sig-multicluster/README.md

```diff
@@ -24,17 +25,59 @@ A Special Interest Group focussed on solving common challenges related to the  m
 * [Mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-multicluster)
 * [Open Community Issues/PRs](https://github.com/kubernetes/community/labels/sig%2Fmulticluster)
 
-## Subprojects
+# Subprojects
 
-The following subprojects are owned by sig-multicluster:
-- **cluster-registry**
-  - Owners:
-    - https://raw.githubusercontent.com/kubernetes/cluster-registry/master/OWNERS
-- **federation**
-  - Owners:
-    - https://raw.githubusercontent.com/kubernetes/federation/master/OWNERS
 
-## GitHub Teams
+
+## Federation v2
+Control Plane for newer Multicluster-specific APIs. Discussions are ongoing to focus future development on multi-cluster specific Kubernetes APIs rather than a reimplementation of single-cluster Kubernetes APIs.
+
+|  |  |
+| -------------- |:-------|
+| Recommended for Production | No code yet (discussions only). |
+| General Availability | No. Prototypes only at this point. |
+| Current Level of Activity | Once to twice weekly discussions as part of Federation working group (most active SIG-Multicluster members). |
+| Owner(s) | Working Group |
+| Where to find it? | Code prototypes in development (one example here). Summary of meetings notes and meeting times [available](ttps://docs.google.com/document/d/1v-Kb1pUs3ww_x0MiKtgcyTXCAuZlbVlz4_A9wS3_HXY). |
+
+## Cluster Registry
```

Comment:
Note: we'll have to update the name here when the component is renamed, as seems likely.